admin/resources/js/api/poolCandidatesOperations.graphql

```diff
@@ -55,7 +55,12 @@ fragment poolCandidate on PoolCandidate {
       fr
     }
   },
-  status
+  status,
+  firstName,
+  lastName,
+  email,
+  telephone,
+  preferredLang
```

Comment:
These are part of the User model, so they should be nested under `user { ... }`. After modifying a graphql operation, you should rerun `npm run codegen`; in this case you'll see an error.

Reply:
Fixed